Week03/lesson05/classwork/week3-relaxr-part2/blog.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="description" content="Follow Relaxr's latest news and learn more about our community.">
+	<meta name="keywords" content="Relaxr, Personal Assistant, PA, AI Technology">
+	<title>Relaxr | Blog</title>
+
+	<!-- external CSS links -->
+	<link rel="stylesheet" href="css/reset.css">
+	<link rel="stylesheet" href="css/style.css">
+	<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,600,800" rel="stylesheet" type="text/css">
+</head>
+<body>
+	<div id="container">
+			<header>
+				<div class="clearfix">
+					<h1><a href="./index.html" class="logo active">Relaxr</a></h1>
+					<nav>
+						<a href="./about.html" class="topnav">About</a>
+						<a href="./faq.html" class="topnav">FAQ</a>
+						<a href="./team.html" class="topnav">Team</a>
+						<a href="./contact.html" class="topnav">Contact Us</a>
+						<a class="topnav active">Blog</a>
+					</nav>
+				</div>
+			</header>
+
+			<div class="clearfix">
+				<main>
+					<article>
+						<h2><a href="./blog/how-i-implemented-relaxr-in-2-weeks-and-changed-my-life">How I implemented Relaxr in 2 weeks and changed my life.</a></h2>
+						<img src="./images/blog_photo1.jpg" alt="Rowers">
+						<summary>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</summary>
+						<div class="clearfix">
+							<div class="readmore"><a href="./blog/how-i-implemented-relaxr-in-2-weeks-and-changed-my-life">Read More &gt;</a></div>
```

Comment:
I love that you created proper fake URLs that go with the content!!

That said, if someone is reviewing some initial work for a website you're building for them, they might be confused when clicking the link breaks. Using `href="#"` is perfect for these scenarios, because then clicking the link won't do anything except perhaps scroll you up to the top of the page, which will make it pretty clear to whichever client is reviewing your website that it's a dummy link, a placeholder